dataservice/api/common/enumerations.yaml

```diff
@@ -0,0 +1,165 @@
+## ------------ NOTES ----------- ##
+# 'Unknown' - contributor specified this in source data
+# 'Not Reported' - should be used as the default if no value is specified in
+# the column or if this column doesn't exist in source data
+
+## --------- Questions --------- ##
+# How should values be cased - lower case, proper/title case?
+# What should default value be if Not Reported in source data?
+
+study:
+    release_status:
+        choices:
+            - Pending # Not sure about the choices here
+        default: Pending
+
+study_file:
+    availability:
+        choices:
+            - Immediate Download
+            - Cold Storage
+        default: Immediate Download
+
+participant:
+    gender:
+        choices:
+            - Male
+            - Female
+            - Unknown
+            - Not Reported
+        default: Not Reported
+    race:
+        choices:
+          - White
+          - American Indian or Alaska Native
+          - Black or African American
+          - Asian
+          - Native Hawaiian or Other Pacific Islander
+          - Other
+          - Unknown
+          - Not Reported
+        default: Not Reported
+    ethnicity:
+        choices:
+            - Hispanic or Latino
+            - Not Hispanic or Latino
+            - Unknown
+            - Not Reported
+        default: Not Reported
+
+diagnosis:
+    diagnosis_category:
+        choices:
+            - Structural Birth Defect
+            - Cancer
+            - Other
+            - Unknown
+        default: Other
+
+phenotype:
+    observed:
+        choices:
+            - Positive
+            - Negative
+            - Unknown
+        default: Unknown
+
+outcome:
+    vital_status:
+        choices:
+            - Alive
+            - Dead
+            - Unknown
+            - Not Reported
+        default: Not Reported
+    disease_related:
+        choices:
+            - Yes
+            - No
+            - Unknown
+            - Not Reported
+        default: Not Reported
+
+biospecimen:
+    analyte_type:
+        choices:
+            - DNA
+            - RNA
+            - Unknown
```

Comment:
`analyte_type` can also take these values https://github.com/kids-first/kf-api-dataservice/blob/master/dataservice/util/data_gen/analyte_type.txt (These values are taken from kf-dictionary) ; and some of CBTTC data also have `Blood`, `Tissue`, `Lys`.

Reply:
These `analyte_type` are a mix of the actual analyte (molecule of study e.g. DNA, RNA) and the source (e.g. EBV Immortalized Normal, Blood), processing (e.g. Repli-G (Qiagen)) and preservation methods (e.g. FFPE). I'd suggest that we keep the `analyte_type` as `DNA`, `RNA` and `Other`. 

There are several ways to breakdown the fields if we'd like to capture them.